modelgauge/tests/safe_v1.py

```diff
@@ -324,31 +182,36 @@ def aggregate_measurements(self, items: List[MeasuredTestItem]) -> SafeTestResul
         return SafeTestResult(persona_results=persona_results)
 
 
-TOGETHER_KEY = InjectSecret(TogetherApiKey)
-
 # TODO: Remove version number from 1.0 UIDs.
 #  Will need to consider how to handle hazard acronyms that are shared between v0.5 and v1.0 (e.g. safe-vcr).
 #  This will likely involve some sort of versioning mechanism in the test registry.
+
+# default llama guard annotator, always
+annotators = DefaultAnnotatorSet()
+annotators.configure()
 TESTS.register(
     SafeTestVersion1,
     "safe-dfm-1.0",
     "dfm",
     Locale.EN_US,
     [SafePersonasVersion1.NORMAL, SafePersonasVersion1.SKILLED],
-    TOGETHER_KEY,
+    annotators,
 )
 
-if PRIVATE_ANNOTATORS_AVAILABLE:
-    VLLM_API_KEY = InjectSecret(VllmApiKey)
-    HUGGINGFACE_KEY = InjectSecret(HuggingFaceKey)  # was: os.getenv("HF_TOKEN", "")
+# private annotators, if available
+try:
+    from modelgauge.private_ensemble_annotator_set import EnsembleAnnotatorSet
```

Comment:
Could all of this be moved to private code? If we have to have private stuff in public files, I'd at least like to keep it to the minimum.